Makefile

```diff
@@ -61,6 +63,15 @@ uninstall: ## Uninstall the compiler from DESTDIR
 
 .PHONY: clean
 clean:
-	rm $(LLVM_EXT_OBJ)
-	rm $(O)
+	rm -f $(LLVM_EXT_OBJ)
+	rm -f $(O)
 
+.PHONY: lib_build
+lib_build: clean $(LLVM_EXT_OBJ)
+	mkdir -p bin
+	$(ENV_LIB) $(COMPILER) build $(FLAGS) src/ic.cr -o $(O)
+
+.PHONY: postinstall
+postinstall: lib_build
+	mkdir -p ../../bin
+	cp $(O) ../../bin/
```

Comment:
Meat of the change is in this block, two make recipes to build and place the `ic` binary when installed via shards